atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/schema/KeyValueServiceMigrator.java

```diff
@@ -108,11 +108,6 @@ private void processMessage(String string, Throwable ex, KvsMigrationMessageLeve
      * Drop and create tables before starting migration.
      */
     public void setup() {
-        /*
```

Comment:
I removed this comment since, as far as I can tell, it's completely wrong. We never drop tables from the source KVS. At the moment, `getMigratableTableNames` is a superset of `getCreatableTableNames`, which means that this comment is somewhat true, but that's by chance - they rely on two different constants in `AtlasDbConstants` that may not reflect this in the future.

In other news, this code is dodgy and has a lot of baked in assumptions that aren't immediately obvious when reading it. It depends on tables listed in `AtlasDbConstants` that I can guarantee are not updated while keeping KVS migrations in mind.